ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/StateUpdateComparison.scala

```diff
@@ -6,10 +6,8 @@ package com.daml.ledger.participant.state.kvutils.tools.integritycheck
 import akka.stream.Materializer
 import akka.stream.scaladsl.Sink
 import com.daml.ledger.participant.state.kvutils.tools.integritycheck.IntegrityChecker.ComparisonFailureException
-import com.daml.ledger.participant.state.v1.{Offset, RejectionReason, TransactionId, Update}
-import com.daml.lf.data.Time
-import com.daml.lf.kv.Normalizer
-import com.daml.lf.transaction.CommittedTransaction
+import com.daml.ledger.participant.state.kvutils.tools.integritycheck.UpdateNormalizer.DefaultNormalizers
```

Comment:
I'd rather change the name to suggest that these are actually the normalizers that apply (and must be used) in all cases, always. Perhaps `MandatoryNormalizers` or so.